### PR TITLE
supported Paddle Op list

### PR DESCRIPTION
**Pull request by liubo-intel**
### Details:
 - Add supported Paddle Op list.

### CODE REVIEW

**Comment by liubo-intel** [comment]
Hi, @ceciliapeng2011 @zhangYiIntel @mangguo321 @luo-cheng2021 and other stakeholders: when you are free, please help have a look at this PR for adding supported Paddle Op list. thanks.

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 1]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+Standard PaddlePaddle(paddlepaddle>=2.1)\* Operators:
+
+| Operator Name in PaddlePaddle\*| Limitations|
+| :----------| :----------|
+| adpative_pool2d | Only support 'NCHW' input data_layout |
+| arg_max | No |
+| assign_value | No |
+| batch_norm | No |
+| bilinear_interp | Only supports the same data_layout |
+| bilinear_interp_v2 | Only supports the same data_layout |
```
Comment (zhangYiIntel) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:535]: This same should be NCHW

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 2]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+| leaky_relu | No |
+| log | No |
+| logical_not | No |
+| matmul | No |
+| matrix_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
+| max_pool2d_with_index | No |
+| mul | No |
+| multiclass_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
+| nearest_interp | Only supports the same data_layout |
+| nearest_interp_v2 | Only supports the same data_layout |
```
Comment (zhangYiIntel) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:567]: Same should be NCHW layout

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md (hunk 1 above)
Comment (zhangYiIntel) [reply docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:535]: In ngraph code, I write `TODO support different data_layout`. It means that later we should support data layouts other than default NCHW.   Not the SAME layout...

Comment (zhangYiIntel) [reply docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:535]:
```suggestion
| bilinear_interp_v2 | Only supports the NCHW data_layout |
```

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 3]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+| hard_swish | No |
+| leaky_relu | No |
+| log | No |
+| logical_not | No |
+| matmul | No |
+| matrix_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
+| max_pool2d_with_index | No |
+| mul | No |
+| multiclass_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
+| nearest_interp | Only supports the same data_layout |
```
Comment (zhangYiIntel) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:566]:
```suggestion
| nearest_interp | Only supports the NCHW data_layout |
```

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md (hunk 2 above)
Comment (liubo-intel) [reply docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md]: done, changed  the interp ops' limitation description.

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 4]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+| matrix_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
+| max_pool2d_with_index | No |
+| mul | No |
+| multiclass_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
+| nearest_interp | Only supports the NCHW data_layout |
+| nearest_interp_v2 | Only supports the NCHW data_layout |
+| pad3d | Not supports the 'Circular' mode |
+| pow | No |
+| pool2d | Only supports the NCHW data_layout |
+| range | Only supports 'FP32' output data_type |
```
Comment (mangguo321) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:571]: Range support int32, int64, float32 output data type, should remove the limitation

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 5]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+
+
+## PaddlePaddle\* Supported Operators
+
+Standard PaddlePaddle(paddlepaddle>=2.1)\* Operators:
+
+| Operator Name in PaddlePaddle\*| Limitations|
+| :----------| :----------|
+| adpative_pool2d | Only supports the NCHW data_layout |
+| arg_max | No |
```
Comment (mangguo321) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:531]: arg_max only supports int64 output data type

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md (hunk 4 above)
Comment (liubo-intel) [reply docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md]: Done, removed 'range' Op limitation and add output limitation for 'arg_max'  Op.

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 6]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+| elementwise_add | No |
+| elementwise_div | No |
+| elementwise_max | No |
+| elementwise_min | No |
+| elementwise_mul | No |
+| elementwise_pow | No |
+| elementwise_sub | No |
+| equal | No |
+| expand_v2 | No |
+| fill_constant_batch_size_like | Only supports the 'FP32' data_type |
```
Comment (luo-cheng2021) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:552]: Should remove the limitation

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 7]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+| pow | No |
+| pool2d | Only supports the NCHW data_layout |
+| range | No |
+| relu | No |
+| relu6 | No |
+| reshape2 | No |
+| rnn | Only supports the 'LSTM' Mode |
+| scale | No |
+| shape | No |
+| slice | Not supports the 'tensor' type of 'starts' and 'ends' |
```
Comment (luo-cheng2021) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:578]: Should remove the limitation

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 8]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+| softmax | No |
+| sigmoid | No |
+| split | No |
+| squeeze2 | No |
+| transpose2 | No |
+| unsqueeze2 | Only supports the 'int32_t' input data_type |
```
Comment (luo-cheng2021) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:584]: Should remove the limitation.

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md (hunk 6 above)
Comment (liubo-intel) [reply docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md]: done, updated limitations of 'fill_constant_batch_size_like', 'slice', 'unsqueeze2'

**Review by ceciliapeng2011 — COMMENTED** [review]
@avladimi  @andrew-zaytsev this is framework PaddlePaddle op list. Could you give your review comments? Anyone else do you know should review it?

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (luo-cheng2021) [reply docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md]: There is no limitation and the document already updated.

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 9]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+| hard_sigmoid | No |
+| hard_swish | No |
+| leaky_relu | No |
+| log | No |
+| logical_not | No |
+| matmul | No |
+| matrix_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
+| max_pool2d_with_index | No |
+| mul | No |
+| multiclass_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
```
[the remark this answers is not in the document]
Comment (liubo-intel) [reply docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:565]: Hi, @ceciliapeng2011 : agree that the description in your comment is more specifically, while do you think it maybe too many words for this table content?

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 10]
```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
...
+| nearest_interp | 'NCW' 'NWC' 'NHWC' 'NCDHW' 'NDHWC' data_layout are not supported |
+| nearest_interp_v2 | 'NCW' 'NWC' 'NHWC' 'NCDHW' 'NDHWC' data_layout are not supported |
+| pad3d | 'Circular' mode is not supported |
+| pow | No |
+| pool2d | 'NHWC' data_layout is not supported |
+| range | No |
+| relu | No |
+| relu6 | No |
+| reshape2 | No |
+| rnn | 'SimpleRNN' and 'GRU' Mode are not supported |
```
Comment (avladimi) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:575]:
```suggestion
| rnn | 'SimpleRNN' and 'GRU' modes are not supported |
```

**Review by avladimi — APPROVED** [review]

**Comment by liubo-intel** [comment]
Hi, @avladimi : we've rebased this PR to the latest master branch. if no further modification needed, could you please help merge this PR? tks:)

**Pull request merged**